sdk/communication/Azure.Communication.Administration/src/CommunicationIdentityClient.cs

```diff
@@ -29,16 +29,34 @@ public CommunicationIdentityClient(string connectionString, CommunicationIdentit
                   options ?? new CommunicationIdentityClientOptions(),
                   ConnectionString.Parse(AssertNotNull(connectionString, nameof(connectionString))))
         { }
-
+        /// <summary> Initializes a new instance of <see cref="CommunicationIdentityClient"/>.</summary>
+        /// <param name="tokenCredential">Token acquired from the Azure Communication Services to be used by service calls</param>
```

Comment:
The token doesn't come from ACS. Let's keep it simple and get inspiration from BlobClient or KeyClient.

```suggestion
        /// <param name="tokenCredential">The TokenCredential used to authenticate requests, such as DefaultAzureCredential.</param>
```

Reply:
Updated!